src/main/io/dji_osd_symbols.h

```diff
@@ -136,17 +139,19 @@
 #define DJI_SYM_WATT                    0x57  // 0x57 is 'W'
 
 // Time
-#define DJI_SYM_ON_H                    0x70
-#define DJI_SYM_FLY_H                   0x71
 #define DJI_SYM_ON_M                    0x9B
 #define DJI_SYM_FLY_M                   0x9C
 
 // Speed
+#define DJI_SYM_SPEED                   0x70
```

Comment:
I think this is RPM rather than speed. But I guess either works.

Reply:
From betaflight source code:
![image](https://github.com/user-attachments/assets/ba13440a-e8d3-4620-981d-a189c2bf040f)